client/app/telecom/pack/xdsl/access/comfortExchange/pack-xdsl-access-comfort-exchange.html

```diff
@@ -0,0 +1,84 @@
+<div class="widget-presentation">
+
+    <h2 class="widget-presentation-title" data-translate="xdsl_access_comfort_exchange_title"></h2>
+
+    <tuc-toast-message></tuc-toast-message>
+
+    <div data-ng-if="$ctrl.isRMAOpened" class="mb-3">
+        <span class="row" data-translate="xdsl_access_comfort_exchange_already_opened_rma"></span>
+        <div data-ng-repeat="rma in $ctrl.rmas track by $index">
+            <div class="row">
+                <span class="col-sm-3 oui-tile__term" data-translate="xdsl_access_comfort_exchange_rma_status"></span>
+                <span class="oui-tile__description" data-ng-bind="rma.status"></span>
```

Comment:
https://github.com/twbs/bootlint/wiki/E013

`Only columns (.col-*-*) may be children of .rows`

:)